src/components/Toast/index.tsx

```diff
@@ -77,6 +77,7 @@ const ToastItem = ({
         cancelAnimationFrame(animationFun)
       }
     }
+    // eslint-disable-next-line
```

Comment:
need delete ?

Reply:
no, keep it here for bypass eslint first for useEffect dependence problem in ToastItem, which would be fix in other PR

Reply:
OK